understanding/error-identification.html

```diff
@@ -55,161 +53,132 @@
       <div class="default-grid with-gap leftcol">
          <aside class="box nav-hack sidebar standalone-resource__sidebar ">
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.3.1:</span>Error Identification (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.3.1:</span>エラーの特定 (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>If an <a href="#dfn-input-error">input error</a> is automatically detected, the item that is in error is identified and the error
-                     is described to the user in text.
+                  <p><a href="#dfn-input-error">入力エラー</a>が自動的に検出された場合は、エラーとなっている箇所が特定され、そのエラーが利用者にテキストで説明される。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Users know an error exists and what is wrong.</dd>
+                  <dt>目標</dt>
+                  <dd>利用者は、エラーが存在し、何が間違っているかを知ることができる。</dd>
```

Comment:
「エラーが存在し、」というのが少し浮いているような感じがします。「エラーが存在することを知ることができる」という話ですね。

この達成基準はエラーの特定であって修正方法の提示までは求めていないので、それをふまえると「何が間違っているか」というより「間違っているのが何なのか」という方が良いかもしれません。

ということで、こうするのはどうでしょうか?

```suggestion
                  <dd>利用者は、エラーが存在することと、間違っているのが何なのかを知ることができる。</dd>
```